### ISSUE TITLE
MSI installer is broken for 8.0.0 due to invalid settings

### ISSUE THREAD

**Issue by jkakavas** [elastic/windows-installers#433]
Default installation adds the following settings: 
```
   transport.tcp.port: 9300
   node.data: true
   node.ingest: true
   node.master: true
   node.max_local_storage_nodes: 1
```

all of which were deprecated in 7.x and removed in 8.0.0

`max_local_storage_nodes` can be removed. For roles we should use the `node.roles` setting. `transport.tcp.port` has been replaced by `transport.port`. 

When doing this change, we should also do our due diligence that all possible settings set by the installer are valid in 8.0.0

**Comment by nkammah** [comment]
@jkakavas are you still seeing the issue ?

**Comment by jkakavas** [comment]
Haven't had the chance to investigate, there hasn't been another beta1 BC so I'd need to get hold of the SNAPSHOT MSI installer and validate, I'll do that over the next hours

**Comment by Mpdreamz** [comment]
This should be fixed now, backported today to `8.0` so should make its way to the next rc1 bc.
